Project.toml

```diff
@@ -4,6 +4,10 @@ authors = ["Charles Kawczynski <kawczynski.charles@gmail.com>"]
 version = "0.4.0"
 
 [deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+EnsembleKalmanProcesses = "aa8a2aa5-91d8-4396-bcef-d4f2ec43552d"
```

Comment:
Is it necessary that we add all of these dependencies?

Reply:
So we have file readers from the clima side, (which don't need these) and file readers from the calibration side (which do). This is because the code is effectively this: 
```f("text_defining_EKP_prior") = instantiation_of_an_EKP_prior.``` 
Currently all the code for the priors is in EKP - so for now we would have to. 

The alternative, would be to move the code to read and write toml files for calibration purposes out to EKP, but we have written this code here for now. What do you think?

Reply:
I think I'd prefer option 1, @simonbyrne, thoughts?